src/screeningsFromAPI.js

```diff
@@ -0,0 +1,13 @@
+export async function getUpcomingScreenings(cmsAdapter) {
+  const screenings = await cmsAdapter.loadAllScreenings();
+  const startPageScreenings = [];
+  const today = new Date();
+  screenings.map((screening) => {
+    const screening_date = new Date(screening.attributes.start_time);
+    const differenceInHours = (screening_date - today) / 3600000; // kan göras med GetDate?
+    if (differenceInHours > 0 && differenceInHours < 120)
+      startPageScreenings.push(screening);
```

Comment:
I recommend to always use curly braces in if statements, even if the "block" is just a single line. Writing if statements like this way can lead to problems, as exemplified in this Stack Overflow thread: https://stackoverflow.com/a/2125078